src/ConstructiveSolidGeometry/SurfacePrimitives/Rectangle.jl

```diff
@@ -0,0 +1,104 @@
+struct Rectangle{T,TN,TL,TW} <: AbstractSurfacePrimitive{T}
+    normal::TN
+    l::TL
+    w::TW
```

Comment:
`l` will be the coordinate that comes alphabetically before `w`,
i.e. `l ≡ x, w ≡ y`, `l ≡ x, w ≡ z` or `l ≡ y, w ≡ z`?

Reply:
Yes!
- RectangleX: `l ≡ y, w ≡ z`
- RectangleY: `l ≡ x, w ≡ z`
- RectangleZ: `l ≡ x, w ≡ y`